### ISSUE TITLE
parisdescartes2 - looking time error?

### ISSUE THREAD

**Issue by melsod** [langcog/mb1-analysis#159]
My RA noticed that a bunch (all?) of the trial looking time data from parisdescartes2 is listed as 18 s in the 03 processed data. Need to track down if this is an error in our R code or something mucked up in the original data file.

**Comment by jkosie** [comment]
It looks like that's their 'total_trial_time' that is 18s for all entries, not 'looking_time'. This would be right if the experiment was fixed trial length rather than infant controlled, wouldn't it? Do they report this info in the lab questionnaires?

**Comment by melsod** [comment]
Yes, that would be correct. But it's showing up in the looking time column (see in the 02 data column F), which wouldn't make sense.

**Comment by jkosie** [comment]
I guess I'm looking in the wrong place. Where is 03 processed data?
